abs.yaml

```diff
@@ -0,0 +1 @@
+aggregate_check: false
```

Comment:
Why's this required out of interest?

Reply:
yes, have the same question, why ?

Reply:
It's a new feedstock and not pinned yet to aggregate. I could have added a pin instead.